pkg/utils/string.go

```diff
@@ -17,3 +20,15 @@ func ParseFloat(str string, defaultValue float64) (float64, error) {
 	}
 	return strconv.ParseFloat(str, 64)
 }
+
+// parsePercentage parse the percent string value
+func ParsePercentage(input string) (float64, error) {
```

Comment:
Can we use a more generic way to define this function, such as ParseFloatWithRatio？